src/main.rs

```diff
@@ -139,6 +144,19 @@ fn main() {
                 .required(false)
                 .requires("config-file")
         )
+        .arg(
+            Arg::with_name("logs-path")
```

Comment:
I agree with @dhrgit  especially since we have a bunch of other arguments representing paths (i.e `chroot_base`, `exec_file`) which do not have "path" in their name and we should at least be consistent.

Reply:
Agree, I renamed them.